src/unstract/sdk/exceptions.py

```
@@ -97,3 +97,12 @@ class FileOperationError(SdkError):
         "Please check specific storage error for "
         "further information"
     )
+
+
+class RetrievalError(SdkError):
```

NIT: @harini-venkataraman I think that this can also be moved to the `prompt-service`. We don't seem to raise this anywhere in this PR. Ignore this comment if its already used somewhere in the SDK. I hope we aren't suppressing the actual error and returning this instead at such places

---

Missed it while refactor. Thanks